WordPress/Classes/ViewRelated/Stats/Shared Views/Date Chooser/SiteStatsTableHeaderView.swift

```diff
@@ -0,0 +1,82 @@
+import UIKit
+
+class SiteStatsTableHeaderView: UITableViewHeaderFooterView, NibLoadable {
+
+    // MARK: - Properties
+
+    @IBOutlet weak var dateLabel: UILabel!
+    @IBOutlet weak var backArrow: UIImageView!
+    @IBOutlet weak var forwardArrow: UIImageView!
+    @IBOutlet weak var bottomSeparatorLine: UIView!
+
+    static let height: CGFloat = 44
+    private typealias Style = WPStyleGuide.Stats
+
+
+    // MARK: - View
+
+    override func awakeFromNib() {
+        applyStyles()
```

Comment:
I believe we need to call from `super.awakeFromNib()` here.

The docs suggest the following:
```
You must call the `super` implementation of `awakeFromNib` to give parent classes the opportunity to perform any additional initialization they require. Although the default implementation of this method does nothing, many UIKit classes provide non-empty implementations. You may call the `super` implementation at any point during your own `awakeFromNib` method.
```

Reply:
🤦‍♂️ I usually catch those _before_ creating the PR. Thanks for noticing! And fixed.